### PR TITLE
Add readme files in the Sample Applications examples folders

### PR DESCRIPTION
**Pull request by DinkoK**

### CODE REVIEW

File: Sample Applications/HotelApp/Description.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+## Environment
+<table>
+	<tr>
+		<td>Product Version</td>
+		<td>2023.3.1113</td>
```
Comment (icom85) [inline Sample Applications/HotelApp/Description.md:5]: The latest version is **2023.3.1114**

File: Sample Applications/ERP/Description.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+## Environment
+<table>
+	<tr>
+		<td>Product Version</td>
+		<td>2023.3.1113</td>
```
Comment (icom85) [inline Sample Applications/ERP/Description.md:5]: The latest version is **2023.3.1114**

File: Sample Applications/MedicalApp/Description.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+## Environment
+<table>
+	<tr>
+		<td>Product Version</td>
+		<td>2023.3.1113</td>
```
Comment (icom85) [inline Sample Applications/MedicalApp/Description.md:5]: The latest version is **2023.3.1114**

File: PdfViewer/CustomDecoder/CustomDecoderCS/CustomDecoder/CustomDecoder.csproj [hunk 4]
```diff
@@ -53,25 +58,42 @@
     <Reference Include="System.Drawing" />
     <Reference Include="System.Windows.Forms" />
     <Reference Include="System.Xml" />
-    <Reference Include="Telerik.WinControls, Version=2020.2.512.40, Culture=neutral, PublicKeyToken=5bb2a467cbec794e, processorArchitecture=MSIL">
-      <HintPath>..\..\..\..\..\..\Program Files (x86)\Progress\Telerik UI for WinForms R2 2020\Bin40\Telerik.WinControls.dll</HintPath>
-      <Private>True</Private>
+    <Reference Include="Telerik.WinControls, Version=2023.3.1114.40, Culture=neutral, PublicKeyToken=5bb2a467cbec794e, processorArchitecture=MSIL">
```
Comment (icom85) [inline PdfViewer/CustomDecoder/CustomDecoderCS/CustomDecoder/CustomDecoder.csproj:61]: Can we use NuGet?

**Review by icom85 — COMMENTED** [review]
Can we check if there is a NuGet with the FreeImage dll and use it if available?

File: PdfViewer/CustomDecoder/CustomDecoderCS/CustomDecoder/CustomDecoder.csproj (hunk 4 above)
Comment (DinkoK) [reply PdfViewer/CustomDecoder/CustomDecoderCS/CustomDecoder/CustomDecoder.csproj:80]: Added

File: Sample Applications/MedicalApp/Description.md (hunk 3 above)
Comment (DinkoK) [reply Sample Applications/MedicalApp/Description.md]: Done

File: Sample Applications/ERP/Description.md (hunk 2 above)
Comment (DinkoK) [reply Sample Applications/ERP/Description.md]: Done

File: Sample Applications/HotelApp/Description.md (hunk 1 above)
Comment (DinkoK) [reply Sample Applications/HotelApp/Description.md]: Done

**Review by icom85 — APPROVED** [review]

**Pull request merged**